src/components/Categories/MonthlyGoodModule.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<div class="tw-relative">
```

Comment:
Since this module is the full width of the screen, on very wide screens, the image will zoom in and look very bad, let's give this a max width so that on ultra wide screens it is just centered with a max width.